### DISCUSSION TITLE
Exporting multiple projects at the same time / batch

### DISCUSSION THREAD

**Discussion by kingfoxii** [mifi/lossless-cut#1445] [Q&A]
Hi,

i would like to know, if it's possible. That i creat 30 Projects where i set up my cuts and than let them run in a single export, creating me all the single files that i cut from the videos?

Thanks in advance for your support.

**Comment by mifi** [comment]
unfortunately not possible yet. See #89

**Reply by kingfoxii** [reply]
Is it possible to start the program more than once? I can only open it once.

**Reply by mifi** [reply]
yes, see https://github.com/mifi/lossless-cut/blob/master/cli.md
